pkg/move/cmd.go

```diff
@@ -105,6 +123,40 @@ func simpleCopy(fs afero.Afero) error {
 	return nil
 }
 
+func filterFilesByTechnology(fs afero.Afero, source string, technologies []string) ([]string, error) {
+	manifestPath := filepath.Join(source, "manifest.json")
+
+	manifestFile, err := fs.ReadFile(manifestPath)
+	if err != nil {
+		return nil, errors.WithMessage(err, "failed to open manifest.json")
+	}
+
+	var manifest Manifest
+	if err := json.Unmarshal(manifestFile, &manifest); err != nil {
+		return nil, errors.WithMessage(err, "failed to parse manifest.json")
+	}
+
+	var paths []string
+
+	for _, tech := range technologies {
+		techData, exists := manifest.Technologies[tech]
+		if !exists {
+			logrus.Warnf("technology %s not found", tech)
+			continue
+		}
+
+		for _, files := range techData {
+			logrus.Infof("processing technology %s", tech)
+
+			for _, file := range files {
+				paths = append(paths, filepath.Join(source, file.Path))
+			}
+		}
+	}
+
+	return paths, nil
+}
+
 func copyFolder(fs afero.Fs, from string, to string) error {
```

Comment:
I wouldn't mess with this function that much, it should be a simple "copy A to B"

you could (probably) just loop over you filtered paths and copy each(ie.: just call this func, unaltered, for each path), because when I looked at an actual `manifest.json` you get every filepath you need and not every folder (or at least it seems like it)